src/scanner.rs

```
@@ -1470,7 +1474,7 @@ impl<T: Iterator<Item = char>> Scanner<T> {
     fn decrease_flow_level(&mut self) {
         if self.flow_level > 0 {
             self.flow_level -= 1;
-            self.simple_keys.pop().unwrap();
```

Removing this unwrap would silence a potential error in the scanner logic.